marklogic-data-hub/src/test/ml-modules/root/test/suites/com.marklogic.hub.impl/generate-indexes.sjs

```diff
@@ -0,0 +1,124 @@
+const test = require("/test/test-helper.xqy");
```

Comment:
With marklogic-unit-test version 1.0 you don't need to have a `com.marklogic.hub.impl` directory, you can actually use directories to organize your test suites.

Not sure if you want to reproduce the entire `com/marklogic/hub/impl directory` structure in `test/suites`, it may be fine to start with `hub/impl` to see how that goes.